src/AltinnCore/Runtime/workflow.bpmn

```diff
@@ -0,0 +1,59 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this file added here?

Reply:
Removed and the workflowdata is read from workflowtemplate path now